src/BannerWithLink/index.tsx

```diff
@@ -22,25 +17,22 @@ export type BrazeMessageProps = {
 };
 
 export type Props = {
-    logButtonClickWithBraze: BrazeClickHandler;
-    submitComponentEvent: (componentEvent: OphanComponentEvent) => void;
-    ophanComponentId?: string;
     brazeMessageProps: BrazeMessageProps;
+    trackClick: TrackClick;
 };
 
 const BannerWithLink: React.FC<Props> = (props: Props) => {
     const {
-        logButtonClickWithBraze,
-        submitComponentEvent,
         brazeMessageProps: {
+            ophanComponentId = COMPONENT_NAME,
```

Comment:
For `BannerWithLink` the `ophanComponentId` is something that we require marketing to provide. It's marked as required in Storybook and the `canRender` enforces this. So it's probably not needed to provide the default/fallback here.
```suggestion
            ophanComponentId,
```

Reply:
Will remove

Reply:
... Done